packages/utils/src/color/hex.ts

```diff
@@ -30,12 +30,11 @@ export function hex2rgb(hex, out)
  * @param {number} hex - Number in hex (e.g., `0xffffff`)
  * @return {string} The string color (e.g., `"#ffffff"`).
  */
-export function hex2string(hex)
+export function hex2string(hex: number): string
 {
-    hex = hex.toString(16);
-    hex = '000000'.substr(0, 6 - hex.length) + hex;
+    const hexString = hex.toString(16).padStart(6, '0');
```

Comment:
Please don’t use padStart, it breaks our es5 compatibility and we don’t want to polyfill this function.